.github/workflows/test.yml

```diff
@@ -44,7 +45,7 @@ jobs:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
       - name: spec tests
-        run: bundle exec rake
+        run: bundle exec rake spec
```

Comment:
I think this (rake -> rake spec) deserves its own commit. Perhaps even make it part of the best practices PR.